src/Suave/Owin.fs

```diff
@@ -634,8 +634,17 @@ module OwinApp =
         HttpContext.request_ >--> HttpRequest.headers_,
         HttpContext.response_ >--> HttpResult.headers_
 
+      let handleResponse  (rhs : Dictionary<string, string[]>) = 
+        let handleKey key =
```

Comment:
What's your opinion on

```fsharp
let handleKey = function
  | "Set-Cookie" ->
    rhs ...
  | _ ->
    [key, String.con...
```

Asking because I'm curious